dom.html

```diff
@@ -0,0 +1,232 @@
+    <script>
+      const storedExperiences =
+        JSON.parse(localStorage.getItem("experiences")) || [];
+
+      for (const experience of storedExperiences) {
+        displayExperience(experience);
+      }
+
+      document
+        .querySelector(".add-experience")
+        .addEventListener("click", function () {
+          document.querySelector(".experience-form").style.display = "flex";
+        });
+
+      document
+        .querySelector(".add-experience-form")
+        .addEventListener("submit", function (event) {
+          event.preventDefault();
+
+          const companyName = document.querySelector(".company-name").value;
+          const startDate = document.querySelector(".start-date").value;
+          const endDate = document.querySelector(".end-date").value;
+          const description = document.querySelector(".description").value;
+
+          const newExperience = {
+            companyName: companyName,
+            startDate: startDate,
+            endDate: endDate,
+            description: description,
+          };
+
+          storedExperiences.push(newExperience);
+          localStorage.setItem(
+            "experiences",
+            JSON.stringify(storedExperiences)
+          );
+
+          displayExperience(newExperience);
+
+          document.querySelector(".add-experience-form").reset();
+          document.querySelector(".experience-form").style.display = "none";
+        });
+
+      function displayExperience(experience) {
+        const experienceEntry = document.createElement("div");
+        experienceEntry.classList.add("experience-entry");
+        experienceEntry.innerHTML = `
+            <strong>${experience.companyName}</strong>
+            <div>Start Date: ${experience.startDate}</div>
+            <div>End Date: ${experience.endDate}</div>
+            <div>Description: ${experience.description}</div>
+            <button onclick="editExperience(${storedExperiences.indexOf(
+              experience
+            )})">Edit</button>
+            <button onclick="deleteExperience(${storedExperiences.indexOf(
+              experience
+            )})">Delete</button>
+        `;
+
+        document
+          .querySelector(".experiences-list")
+          .appendChild(experienceEntry);
+      }
+
+      function deleteExperience(index) {
+        storedExperiences.splice(index, 1);
+        localStorage.setItem("experiences", JSON.stringify(storedExperiences));
+        updateDisplay();
+      }
+
+      function editExperience(index) {
+        const experience = storedExperiences[index];
+        document.querySelector(".experience-form").style.display = "flex";
+
+        document.querySelector(".company-name").value = experience.companyName;
+        document.querySelector(".start-date").value = experience.startDate;
+        document.querySelector(".end-date").value = experience.endDate;
+        document.querySelector(".description").value = experience.description;
```

Comment:
There is multiple usage of "document.querySelector(".description").value" like all other cases. This needs to go in one variable and must be used through that variable only.

Reply:
fix